### ISSUE TITLE
[Bug]: Incorrect folder name if project uses `nwidart/laravel-modules`

### ISSUE THREAD

**Issue by Oleksandr-Moik** [laravel-idea/plugin#732]
### Bug description

If the project uses the `nwidart/laravel-modules` package for modules, creating files for the **root module** generates the wrong path.

When I want to create a migration, I expect a new file to be created in `/database/migrations`, but it is created in `/Database/Migrations`, just like for a regular module.

![image](https://user-images.githubusercontent.com/50796878/222096229-bbc41868-6350-451d-81ee-467dd859d7fa.png)

When I select for example module **Blog**,the file path is fine

![image](https://user-images.githubusercontent.com/50796878/222098105-a3f0dc85-0a30-4383-9b4a-82c14f4611f8.png)

### Plugin version

6.3.1.223

### Operating system

Linux

### Steps to reproduce

Install package `nwidart/laravel-modules:^8.3` for laravel 8 project.

Open code generation. Select create migration and select **Root App module** in Module dropdown.
 
![migration create modal](https://user-images.githubusercontent.com/50796878/222095615-155f4950-12fd-4595-bd34-cf0999a71b4b.png)

Confirm and new migration will be created in `/Database/Migrations`

![image](https://user-images.githubusercontent.com/50796878/222096229-bbc41868-6350-451d-81ee-467dd859d7fa.png)

**Comment by adelf** [comment]
Hi. Sadly, you reported right after releasing a new version. I tried to fix it, but I can't test it in Linux(this bug only happens there). 

Can you test it if I build a special fix-release for you?

**Comment by Oleksandr-Moik** [comment]
Yes🙂, I can install and test

**Comment by adelf** [comment]
Please download https://laravel-idea.com/releases/Laravel%20Idea-6.3.1.223.zip
Then in PhpStorm: Settings > Plugins > Little cogwheel > Install from file.

**Comment by Oleksandr-Moik** [comment]
Done. What`s next?

----

But before install, I remembered that the plugin has settings for code generation. Maybe because of this the path for generation is the same and does not depend on the presence of modules?

![image](https://user-images.githubusercontent.com/50796878/222392687-d84eca7a-1a1a-4ec7-ab1d-f5d196f929f6.png)

Because a similar story with the creation of factories

![image](https://user-images.githubusercontent.com/50796878/222394223-d4c1b6b2-5960-45c9-abda-c96833129780.png)
![image](https://user-images.githubusercontent.com/50796878/222394439-ca76021d-a2d0-4d9d-a62a-b979f7544ee2.png)

----

It seems more difficult to add support for this than to move a class/file with F6 key 😅

**Comment by adelf** [comment]
No) It shouldn't.
In this release, I implemented a new directory search logic - case insensitive. So, it finds the first directory with case-insensitive equals with the needed one. Or creates new.
```
fun PsiDirectory.findSubDirectoryCaseInsensitive(name: String): PsiDirectory? {
    val lowerCasedName = name.lowercase()

    return subdirectories.firstOrNull {
        it.name.lowercase() == lowerCasedName
    }
}
```

**Comment by Oleksandr-Moik** [comment]
Cool! Migrations and factories are created in the correct directory. 

But I had to change the folder path for the factories (from previous comment) from `database/factories` to `Database/Factories`, because the wrong namespace was generated.

**Comment by adelf** [comment]
Thanks. I'll fix that too.

**Comment by adelf** [comment]
Fixed in 7.0
